### ISSUE TITLE
Color for Tables

### ISSUE THREAD

**Issue by Salompas** [hlissner/emacs-doom-themes#316]
When previewing Latex tables in org-mode, the default font color is black.
This is not an issue when previewing tables when using a light theme, like `doom-solarized-light`:
<img width="1280" alt="Screen Shot 2019-07-31 at 1 30 21 PM" src="https://user-images.githubusercontent.com/7663115/62233820-5b9d9b80-b397-11e9-9f9f-f18ce7cf10ac.png">

However, with dark themes, like `doom-one`, the previews become very hard to read:
<img width="1280" alt="Screen Shot 2019-07-31 at 1 31 26 PM" src="https://user-images.githubusercontent.com/7663115/62233890-80920e80-b397-11e9-80ea-441a92bf324c.png">

The color issue is only happening with table previews, but not with Latex equations.
In other words, previewing equations (like $x+y$) on dark themes uses a white font, and on light themes it uses a dark font. For some reason, the same thing does not happen with tables.

Is it possible to fix this color issue?

Related to this [stack post](https://emacs.stackexchange.com/questions/42395/color-when-visualizing-latex-table-in-orgmode).

**Comment by hlissner** [comment]
Sorry for the tremendously late response: I've been using this for a while, on the latest version of org:

```elisp
(with-eval-after-load 'org
  (setq org-format-latex-options
        (plist-put org-format-latex-options
                  :scale 1.5   ; larger previews
                  :foreground 'auto  ; match the theme foreground
                  :background 'auto)))
```

Does this not work for you?
